development/src/utility/hexagons.js

```diff
@@ -666,6 +645,36 @@ var HexGrid = Class.create( {
 		});
 	},
 
+	/* getHexLine(x, y, dir, flipped)
```

Comment:
Did the correction myself and testing; might have bumped into another bug hmm

I meant the initial code that you copy-pasted instead of making good use of it as it was.